assets/prototype/domain/eventEditor/hooks/useDatetimeContext.ts

```diff
@@ -0,0 +1,12 @@
+import { useContext } from 'react';
+
+import { DateTimeContext, DateTimeContextProps } from '../context/DateTimeProvider';
+
+const useDatetimeContext = (): DateTimeContextProps => {
+	const value = useContext<DateTimeContextProps>(DateTimeContext);
+	if (!value) {
+		throw new Error('useDatetimeContext must be used inside <DateTimeProvider> component');
```

Comment:
This looks like an dev-related error.
Maybe we might consider using `invariant`: https://github.com/zertosh/invariant

Reply:
I haven't used that before.